pkg/Microsoft.Private.CoreFx.UAP/Microsoft.Private.CoreFx.UAP.pkgproj

```diff
@@ -17,101 +17,27 @@
 
     <NETStandardVersion>2.0</NETStandardVersion>
```

Comment:
Did you want to move this to the common targets as well? It looks to only be used in the VerifyNETStandard target.